app/src/test/java/org/greenstand/android/TreeTracker/capture/TreeCaptureViewModelTest.kt

```diff
@@ -0,0 +1,124 @@
+package org.greenstand.android.TreeTracker.capture
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import io.mockk.*
+import io.mockk.impl.annotations.MockK
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import org.greenstand.android.TreeTracker.MainCoroutineRule
+import org.greenstand.android.TreeTracker.models.SessionTracker
+import org.greenstand.android.TreeTracker.models.TreeCapturer
+import org.greenstand.android.TreeTracker.models.UserRepo
+import org.greenstand.android.TreeTracker.models.location.LocationDataCapturer
+import org.greenstand.android.TreeTracker.models.user.User
+import org.greenstand.android.TreeTracker.usecases.CreateFakeTreesParams
+import org.greenstand.android.TreeTracker.usecases.CreateFakeTreesUseCase
+import org.greenstand.android.TreeTracker.utils.emptyUser
+import org.greenstand.android.TreeTracker.utils.getOrAwaitValueTest
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+@ExperimentalCoroutinesApi
+class TreeCaptureViewModelTest {
+
+    @get:Rule
+    var instantTaskExecutorRule = InstantTaskExecutorRule()
+
+    @get:Rule
+    var mainCoroutineRule = MainCoroutineRule()
+
+    private val profilePicUrl = ""
+
+    @MockK(relaxed = true)
+    private lateinit var userRepo: UserRepo
+
+    @MockK(relaxed = true)
+    private lateinit var treeCapturer: TreeCapturer
+
+    @MockK(relaxed = true)
+    private lateinit var sessionTracker: SessionTracker
+
+    @MockK(relaxed = true)
+    private lateinit var createFakeTreesUseCase: CreateFakeTreesUseCase
+
+    @MockK(relaxed = true)
+    private lateinit var locationDataCapturer: LocationDataCapturer
+
+    private lateinit var treeCaptureViewModel: TreeCaptureViewModel
+
+    @Before
+    fun setUp() {
+        MockKAnnotations.init(this)
+        coEvery { userRepo.getPowerUser() } returns emptyUser
+
+        treeCaptureViewModel = TreeCaptureViewModel(
+            profilePicUrl = profilePicUrl,
+            userRepo = userRepo,
+            treeCapturer = treeCapturer,
+            sessionTracker = sessionTracker,
+            createFakeTreesUseCase = createFakeTreesUseCase,
+            locationDataCapturer = locationDataCapturer
+        )
+    }
+
+    @Test
+    fun isFirstTrack() = runBlocking {
```

Comment:
Please use the `` syntax and describe in a bit more detail what the method is testing. For example:

fun `WHEN tree count greater than 0 THEN firstTrack is true`() {
    ... your code here
}

Also, this tests when it's greater than one, but this means we should also have a test confirming it's true if the tree count is less than 1.